spy-reaper/spy-reaper.py

```diff
@@ -45,7 +45,13 @@ def main():
     client_app.connect("127.0.0.1", port, clientId=0)
     sys.stderr.write("serverVersion:{0}\nconnectionTime:{1}\n".format(client_app.serverVersion(), client_app.twsConnectionTime()))
     sys.stderr.write("Handling incoming messages.\n")
+    server_ver_req = LooseVersion("124")
+    server_ver = LooseVersion(str(client_app.serverVersion()))
+    if server_ver < server_ver_req:
+        sys.stderr.write("Server version required is '124', server version installed '{0}' \n".format(server_ver))
```

Comment:
Substitute server_ver_req in place of 124 in the error message please.  That way we only have one hard-coded string literal.